components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/tests/Services/GetSimpleEventFilterTests.cs

```diff
@@ -10,13 +10,11 @@ namespace Microsoft.Azure.IIoT.OpcUa.Protocol.Tests.Services {
     using Moq;
```

Comment:
This class is in a different namespace than the other tests. Let's keep using `Microsoft.Azure.IIoT.OpcUa.Protocol.Services` for consistency.

Reply:
Fixed